skype/skype-ps/skype/Remove-CsHuntGroup.md

```diff
@@ -148,31 +167,14 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -AsJob
-PARAMVALUE: SwitchParameter
-
-```yaml
-Type: SwitchParameter
-Parameter Sets: (All)
-Aliases: 
-Applicable: Skype for Business Online
-
-Required: False
-Position: Named
-Default value: None
-Accept pipeline input: False
-Accept wildcard characters: False
-```
-
 ### CommonParameters
 This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
 
 ## INPUTS
-### System.Uri
-Represents the SIP uri of a call queue.
 
 ## OUTPUTS
-### Microsoft.Skype.Rgs.Configuration.Huntgroup
+
+### Microsoft.Rtc.Management.Hosted.HuntGroup.RemoveCsHuntGroupCmdlet
```

Comment:
Should be:
Microsoft.Rtc.Management.Hosted.HuntGroup.Models.HuntGroup

Reply:
Fixed